### PR TITLE
improve instructions

### PR DESCRIPTION
**Pull request by oliviertassinari**
Another iteration on the challenge. All-in-all, I love it, excited to see how it will perform!

Preview: https://github.com/mui/tech-challenge-design-engineer/blob/f9c875f758a8ad01ef2678c4acaa96266a3c7280/README.md. 

I have done:

- Force port 3002 per https://www.notion.so/mui-org/Ports-b113d0d4c76c4db9a0ef99c703d0084c?pvs=4#a1c936d8f6324c4f9030a1b5be99d54e so reviewers can quickly run it.
- Created a dedicated notion page for this challenge https://www.notion.so/mui-org/tech-challenge-design-engineer-146259ac58fa460cbe131a7db81ebf47
- Fix `public/snackbar-reference.gif` to get 60 FPS and 200 PPI. The reference was burning my eyes 😁

https://user-images.githubusercontent.com/3165635/229311469-ba2a5b1d-f961-4042-ab43-d6996d27b32b.mov

I have applied https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917?pvs=4#9c47cb10d26e4a95be23b01fdb96f431

- Normalized the description of the challenge with the two others, effectively, taking the best out of this one.
- Renamed the challenges to prefix them with `tech-challenge-`. I was hesitating about merging the 3 repositories into a single one, but in the end, I didn't. This new prefix alone helps a lot to apply mass changes between the challenge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,27 +19,41 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
 
-## Challenge summary
+## Introduction
 
 Given that the Design Engineer position sits right at the intersection between engineering and design, this challenge was tailored to elevate your skills in both areas.
 Our challenge is to **develop a stacked Snackbar** component.
 
 Here's a reference design:
 
-![Snackbars example design](public/snackbar-reference.gif)
```
Comment (oliviertassinari) [inline README.md:29]: ⚠️ This should have been:

```suggestion
![Snackbars example design](./public/snackbar-reference.gif)
```

to reference the file in the current commit, not in the default branch.

File: README.md [hunk 2]
```diff
@@ -19,27 +19,41 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
+Note that it's only an example.
+We encourage you to add your creative spin to the design, we want to see your design taste!
+
+## Objective
 
-Note that's just an example, definitely feel encouraged to add your own creative spin to the design, we want to see your design taste!
 We do have a few requirements, though:
 
-- **Stack:** use MUI Base’s [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or even other headless/unstyled component libraries as the foundation for it.
+- **Stack:** You can use Base UI's [`UnstyledSnackbar`](https://mui.com/base/react-snackbar/) component or another headless component library as the foundation for it.
```
Comment (oliviertassinari) [inline README.md:38]: ~MUI Base~ -> Base UI

File: README.md [hunk 3]
```diff
@@ -19,27 +19,41 @@ At the end of the day, it's about [_writing less code_](https://youtu.be/GnO7D5U
...
-## Challenge summary
+## Introduction
 
 Given that the Design Engineer position sits right at the intersection between engineering and design, this challenge was tailored to elevate your skills in both areas.
 Our challenge is to **develop a stacked Snackbar** component.
 
 Here's a reference design:
 
-![Snackbars example design](public/snackbar-reference.gif)
+<img alt="Snackbars example design" src="./public/snackbar-reference.gif" width="465" height="392" />
```
Comment (danilo-leal) [inline README.md:29]: That's so much better ⎯ thanks! How did you record this git, just out of curiosity?

**Review by danilo-leal — APPROVED** [review]
Looks great!

File: README.md (hunk 3 above)
Comment (oliviertassinari) [reply README.md:29]: > How did you record this git, just out of curiosity?

I used this: https://www.notion.so/mui-org/Blog-247ec2bff5fa46e799ef06a693c94917?pvs=4#39223739d754407d8c1d6edb9786d8bd

Comment (danilo-leal) [reply README.md:29]: Thank you!

**Pull request merged**